### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AbdulazizAlhasan**
Test update

# Your Details

- Your Name: Abdulaziz Alhasan
- Your City: London
- Your Slack Name: Abdulaziz Alhasan

# Homework Details

- Module: HTML-CSS- courseeork
- Week: 2

### CODE REVIEW

**Review by LucyMac — COMMENTED** [review]
Nice looking page and good HTML structure -though try and use `section` instead of `div` when you can. 
What are the other 40 files added to the project for - are you using them? You shouldn't need them for this particular task.

File: css/style.css [hunk 1]
```diff
@@ -16,4 +17,181 @@ body {
...
+    }
+
+
+.topSection {
+    width: 100%;
+    height: 750px;
+    margin: 0;
+    background-image: url("../img/first-background.jpg");
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
```
Comment (LucyMac) [inline css/style.css:88]: Watch out, this rule is forcing your image to stretch across the space at all times, which is distorting it. Remove this line and it should work fine 👍

**Comment by AbdulazizAlhasan** [comment]
@LucyMac  I have updated my submission to include you your comments, also added the new section as suggested. Any feeback is much appreciated.

**Comment by AbdulazizAlhasan** [comment]
https://cyf-abdulazizalhasan-karma.netlify.app/

**Pull request closed without merging**